schema/containers.schema.json

```
@@ -16,7 +16,7 @@
             "kind": {
               "description": "kind of container contents",
               "type": "string",
-              "enum": ["figure", "table"]
+              "enum": ["figure", "table", "blockquote"]
```

I think this should be `quote`? As we are supporting pull-quotes, epigraphs, and blockquotes? Maybe just a typo!?

```suggestion
              "enum": ["figure", "table", "quote"]
```

---

Hmm, I got my wires crossed here - thanks. I was thinking about the child `type`, not container `kind`.